optimum/intel/ipex/modeling_base.py

```diff
@@ -144,11 +146,14 @@ def __init__(
             self._device = torch.device("cpu")
 
         # CPU only support jit model for now.
-        if not isinstance(model, torch.jit.RecursiveScriptModule):
-            config = model.config if config is None else config
-            use_cache = getattr(model.config, "use_cache", False)
-            model = ipex_jit_trace(model, self.export_feature, use_cache)
-            config.torchscript = True
+        if export:
+            if isinstance(model, torch.jit.RecursiveScriptModule):
+                logger.warning("The model has been exported already.")
+            else:
+                config = model.config if config is None else config
+                use_cache = kwargs.get("use_cache", None)
```

Comment:
actually should be set by default to True to fix compatibility with v1.17.0

https://github.com/huggingface/optimum-intel/blob/v1.17.2/optimum/intel/ipex/modeling_base.py#L159
```suggestion
                use_cache = kwargs.get("use_cache", True)
```